scripts/linux.sh

```diff
@@ -1,18 +1,21 @@
 #!/bin/bash
 
-function ignore_first_line {
+set -u
+set -e
+
+ignore_first_line() {
   tail -n +2
 }
 
-function get_uuids {
-  /sbin/blkid -s UUID -o value $1*
+get_uuids() {
+  /sbin/blkid -s UUID -o value "$1"*
 }
 
-function get_mountpoint {
-  df --output=source,target | grep "^$1" | awk '!($1="")&&gsub(/^ /,"")'
+get_mountpoint() {
+  mount | grep "^$1" | cut -d ' ' -f3
```

Comment:
Oops, this was a bug fix I had in my branch that I never sent. I'll split to another PR.